src/custom/pages/Swap/hooks/useSwapButtonState.ts

```diff
@@ -12,31 +11,30 @@ import { useExpertModeManager } from '@src/state/user/hooks'
 import TradeGp from 'state/swap/TradeGp'
 
 export enum SwapButtonState {
-  swapIsUnsupported,
-  walletIsUnsupported,
-  wrapError,
-  shouldWrapNativeToken,
-  shouldUnwrapNativeToken,
-  switchToWeth,
-  feesExceedFromAmount,
-  insufficientLiquidity,
-  zeroPrice,
-  transferToSmartContract,
-  fetchQuoteError,
-  offlineBrowser,
-  loading,
-  walletIsNotConnected,
-  readonlyGnosisSafeUser,
-  needApprove,
-  swapDisabled,
-  swapError,
-  expertModeSwap,
-  regularSwap,
+  swapIsUnsupported = 'swapIsUnsupported',
```

Comment:
nit: we usually capitalise enum states

Reply:
Done: https://github.com/cowprotocol/cowswap/pull/1049/commits/c17b9decc6570e42cdea00f039ba335d9116c23c